source/query-bar.txt

```diff
@@ -306,49 +306,23 @@ as favorites. To view the list of queries saved as favorites, click on
 .. figure:: /images/compass/query-history-favorite.png
    :figwidth: 316px
 
-.. class:: hidden
-
-   .. toctree::
-      :titlesonly:
-
-      /export-query-to-language
-
 Limitations
 -----------
 
-Limitations apply to the following query operators:
```

Comment:
I think we still want this sentence. Sorry, was probably unclear in my comment. I was saying we should just remove `query` from this sentence so we say:

`Limitations apply to the following operators:`

Reply:
👍